src/rdo.rs

```diff
@@ -404,6 +410,15 @@ pub fn rdo_mode_decision(
     mode_contexts.push(cw.find_mvrefs(bo, ref_frames, &mut mv_stack, bsize, false, fi, false));
 
     if fi.frame_type == FrameType::INTER {
+      let mut pmv = [MotionVector{ row: 0, col: 0 }; 2];
+      if mv_stack.len() > 0 { pmv[0] = mv_stack[0].this_mv; }
+      if mv_stack.len() > 1 { pmv[1] = mv_stack[1].this_mv; }
+      let cmv = pmvs[ref_slot_set[i] as usize].unwrap();
+      mvs_from_me.push([
+        motion_estimation(fi, fs, bsize, bo, ref_frames[0], cmv, seq.bit_depth, &pmv),
```

Comment:
So, the roles are exchanged now.
The new parameter 'cmv' is for 'spatial prediction', i.e. ME seed, and pmv is from neighboring blocks searched by find_mvrefs() as defined by av1 spec, correct?

Reply:
cmv is now the search center/seed. pmvs are the vectors used to determine the bit cost